config/docker/mongo/mongoDbPrepare.js

```diff
@@ -1,80 +1,90 @@
+print(`DB Prepare`);
```

Comment:
maybe I missed it, where do you specify the database to create the collections in?
There is an env MONGO_INITDB_DATABASE that should be set.

Reply:
You are right, I've just found out that the server inits the database itself, that's why it was working when I pushed the branch. Why is compose creating the tables if the server can do it? We would avoid maintaining the schemas on bootstrap.

Reply:
it would also boot faster...

Reply:
@Jaguar0625 @gimer Thoughts?